.github/workflows/drivers.yml

```diff
@@ -32,7 +32,7 @@ jobs:
     needs: files-changed
     if: github.event.pull_request.draft == false && needs.files-changed.outputs.backend_all == 'true'
     runs-on: ubuntu-22.04
-    timeout-minutes: 60
+    timeout-minutes: 45
```

Comment:
With @metamben 's updates to Oracle tests, we shouldn't need 60 minute timeouts anymore. Even 45 is too much, but until we find a way to optimize them further and to cut time, let's stick with this value.